engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/interop/MetaServicesTest.java

```diff
@@ -0,0 +1,169 @@
+package org.enso.interpreter.test.interop;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.net.URI;
+import java.util.List;
+import java.util.function.Supplier;
+import org.enso.interpreter.node.expression.builtin.meta.LookupServicesNode;
+import org.enso.interpreter.runtime.data.Type;
+import org.enso.interpreter.runtime.data.text.Text;
+import org.enso.interpreter.runtime.error.PanicException;
+import org.enso.test.utils.ContextUtils;
+import org.graalvm.polyglot.PolyglotException;
+import org.graalvm.polyglot.Source;
+import org.hamcrest.MatcherAssert;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+public class MetaServicesTest {
+  @ClassRule public static ContextUtils ctx = ContextUtils.createDefault();
+
+  @Test
+  public void loadFileSystemServices() throws Exception {
+    final URI uri = new URI("memory://services.enso");
+    final Source src =
+        Source.newBuilder(
+                "enso",
+                """
+    import Standard.Base.System.File.File_System_SPI
+    import Standard.Base.Meta
+    spis =
+        Meta.lookup_services File_System_SPI
+    """,
+                "services.enso")
+            .uri(uri)
+            .buildLiteral();
+
+    var arr = ctx.evalModule(src, "spis");
+
+    assertTrue("Got SPIs", arr.hasArrayElements());
+    var len = arr.getArraySize();
+    for (var i = 0L; i < len; i++) {
+      var p = arr.getArrayElement(i);
+      System.err.println("found " + p);
+      if (p.getMember("protocol").asString().equals("enso")) {
+        var type = p.getMember("typ");
+        assertTrue("It is a type", type.isMetaObject());
+        assertEquals("Enso_File", type.getMetaSimpleName());
+        return;
+      }
+    }
+    fail("Not found `enso` file protocol among: " + arr);
+  }
+
+  @Test
+  public void missingConversionYieldsDataflowError() {
+    var arr =
+        ctx.evalModule(
+            """
+    import Standard.Base.System.File.File_System_SPI
+    type Broken_Impl
+
+    main = [File_System_SPI, Broken_Impl]
+    """);
+    var node = new MockLookupServicesNode();
```

Comment:
Testing broken registrations is easier with `MockLookupServicesNode` rather than real `LookupServicesNode`. The real node is using `PackageManager`  & co. and requires the whole Enso infrastructure to get into the _broken state_ that needs to be tested.